packages/rulesets/src/spectral/az-arm.ts

```diff
@@ -585,6 +586,20 @@ const ruleset: any = {
       },
     },
 
+    // RPC Code: RPC-Put-V1-29
+    ConsistentResponseSchemaForPut: {
+      description: "A Put operation must return the same schema for 200 and 201 response codes",
+      message: "{{error}}",
+      severity: "error",
+      stagingOnly: true,
+      resolved: true,
+      formats: [oas2],
+      given: "$.paths.*",
+      then: {
+        function: consistentputresponsebody,
```

Comment:
>consistentputresponsebody [](http://example.com/codeflow?start=18&length=25)

I think it would make things easy to keep the rule and function name the same